packages/repocop/src/rules/repository.ts

```diff
@@ -303,7 +305,10 @@ export async function getAlertsForRepo(
 				direction: 'asc', //retrieve oldest vulnerabilities first
 			});
 
-		return alert.data;
+		const openRuntimeDependencies = alert.data.filter(
+			(a) => a.dependency.scope === 'runtime',
```

Comment:
```suggestion
			(a) => a.dependency.scope !== 'development',
```

as this value could be `null` or `undefined`, we should try to catch those values